Dockerfile

```diff
@@ -1,10 +1,5 @@
-FROM maven as build
-WORKDIR /app
-COPY . .
-RUN mvn install
 
-FROM openjdk:8
-WORKDIR /app
-COPY --from=build /app/target/users.jar app.jar
-EXPOSE 8081
+FROM openjdk:17
+COPY /target/users.jar app.jar
```

Comment:
why did you remove **--from=build** ? 
The --from=build argument is necessary to copy files between different stages in a multi-stage Docker build.